pkgs/development/tools/misc/easypdkprog/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "easypdkprog";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "free-pdk";
+    repo = "easy-pdk-programmer-software";
+    rev = "3a557ccef00feb9ac3aa20c5984c158370f77cd7";
+    sha256 = "06bn86rd57ff567l0ipx38raih0zll3y16lg5fpn7c601a9jldps";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin/
+    cp easypdkprog $out/bin/
+  '';
+
+  postInstall = lib.optionalString stdenv.isLinux ''
+    mkdir -p "$out/etc/udev/rules.d/"
+    cp Linux_udevrules/70-stm32vcp.rules $out/etc/udev/rules.d/
+  '';
+
+  meta = with lib; {
+    description = "read, write and execute programs on PADAUK microcontroller";
+    homepage = "http://openocd.sourceforge.net/";
+    license = licenses.gpl3;
+    # maintainers = with maintainers; [ david-sawatzke ];
```

Comment:
Add yourself as a maintainer to `maintainers/maintainer-list.nix` . We probably do not want to add a derivation, unless it has a maintainer (or is very popular).